### PR TITLE
Add German translations

### PR DESCRIPTION
**Pull request by xstyyy**
Translates every page outside of API documentation to german

### CODE REVIEW

**Comment by takase1121** [comment]
insane

File: locales/de/template.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/svg" href="/assets/img/logo.svg">
+    <link rel="stylesheet" href="/assets/stylesheet.css"/>
+    <script type='text/javascript' src='/assets/javascript.js'></script>
+    <meta name="description" content="Ein leichter, simpler, schneller, funktionsreicher, und extrem erweiterbarerer Texteditor geschrieben in C, und Lua, angepasst von lite.">
+  </head>
+  <body>
+    <header>
+      <logo>
+        <a href='/de'><img class='logo' src='/assets/img/logo.svg'/></a>
```
Comment (takase1121) [inline locales/de/template.html]: Adapt this to use `{{ lang }}` and `{{ path }}` as necessary.

**Review by takase1121 — CHANGES_REQUESTED** [review]
Please rebase your changes with the latest master to support the language selector (don't forget to unhide it).

**Comment by xstyyy** [comment]
i've added the changes. If there's anything I did incorrectly or that I should add, then make sure to tell me

**Comment by takase1121** [comment]
> i've added the changes. If there's anything I did incorrectly or that I should add, then make sure to tell me

Can you unhide the language selector in the EN pages as well?

**Comment by takase1121** [comment]
right, so one more thing.

Since I don't speak German (and I doubt @Guldoman and @adamharrison can), so we won't be able to maintain this. I was partly hoping you would be able to maintain it when contents get updated in the English site, but this is just a problem with i18n.

If you're fine, we might be putting some kind of banner to warn users that latest documentation can be found in the EN version (by checking modtime or something).

**Comment by xstyyy** [comment]
> If you're fine, we might be putting some kind of banner to warn users that latest documentation can be found in the EN version (by checking modtime or something)

sure

**Review by takase1121 — APPROVED** [review]

**Pull request merged**